### ISSUE TITLE
On close orders in the manual

### ISSUE THREAD

**Issue by dservais** [ccxt-dev/ccxt#249]
Closed Orders

exchange.fetchOpenOrders (symbol = undefined, params = {})

Shouldn't it be exchange.fetch**Close**Orders  ??

**Comment by kroitor** [comment]
@dservais hi! 

Sorry, but I did not quite understand the question )

To us it looks like *open* and *closed* are adjectives derived from verbs "to open" and "to close".

So, according to English grammar:
- an open order is *"Open"*
- a closed (completed) order is *"Closed"*
- a canceled order is *"Canceled"*

*Closed* has nothing to do with the distance here. The order is not *"close"* nor it is *"near"*, it is completed, therefore *"Closed"*.

In other words, `fetchOpenOrders()` and `fetchClosedOrders()` are named correctly. Does this answer your question? Let us know if this is not what you were asking about.

**Comment by dservais** [comment]
Actually it is not a question but à typo in the text of the section close orders in your manuel or documentation. 
For the section "open orders" the name of the function is "fetchOpenOrder" but for the section "close orders" it is also written "fetchOpenOrders", From my point of view, the function name should be "fetchCloseOrders". 
Best regards

**Comment by kroitor** [comment]
Oh, I'm sorry, did not get it at first. My bad! Thx for pointing that out, I'll fix that typo in the Manual!

**Comment by kroitor** [comment]
Fixed it, thx again!
